deploy/install.yaml

```diff
@@ -161,6 +161,7 @@ data:
   PYTHONUNBUFFERED: "TRUE"
   IMAGESWAP_MODE: "MAPS"
   IMAGESWAP_LOG_LEVEL: "INFO"
+  IMAGESWAP_DISABLE_AUTO_MWC: "TRUE"
```

Comment:
**question:** do we want `IMAGESWAP_DISABLE_AUTO_MWC: "TRUE"` in this file if the default is meant to be false?

Reply:
I think this was a manual change for testing while I was developing. I'm gonna go back and make sure, but likely I just missed reverting it.

Reply:
Ok, this was leftover from testing, just cleaned it up